Tests/Helpers.fs

```diff
@@ -27,3 +30,13 @@ let makeFakeDocument (content: string) : Doc =
 let makeFakeDocumentLines (lines: array<string>) : Doc =
     let text = Text.mkText (String.concat System.Environment.NewLine lines)
     Doc.mk (PathUri.fromString "memory://fake.md") (PathUri.fromString "memory://") None text
+
+let applyDocumentAction (doc: Doc) (action: DocumentAction)  = 
+    let (before, after) = doc.text.Cutout(action.edit)
+
+    before + action.newText + after
+
```

Comment:
curious: doesn't `make fmt` remove extra newlines?

Reply:
I think that's because you only have it running on the main project: https://github.com/artempyanykh/marksman/blob/main/Makefile#L62

Not going to change in this PR, but can do in the next one and reformat the tests as well.